packages/sommelier.rb

```diff
@@ -459,18 +460,31 @@ def self.postinstall
     else
       ExitMessage.add <<~EOT0.lightblue
 
-        Removed old sommelier environment variables in ~/.bashrc.
-        A backup of the original is stored in ~/.bashrc.#{@now}
+        Removed old sommelier environment variables in: ~/.bashrc
+        A backup of the original is stored in: ~/.bashrc.#{@now}
       EOT0
     end
 
-    FileUtils.touch "#{HOME}/.sommelier.env" unless File.exist? "#{HOME}/.sommelier.env"
+    unless File.exist? "#{CREW_PREFIX}/.config/.sommelier.env"
+      FileUtils.mkdir_p "#{CREW_PREFIX}/.config"
+      if File.exist? "#{HOME}/.sommelier.env"
+        FileUtils.cp "#{HOME}/.sommelier.env", "#{CREW_PREFIX}/.config/.sommelier.env"
+        ExitMessage.add <<~EOT0.lightblue
+
+          Overrides of the default sommelier configuration must now be stored in:
+          #{CREW_PREFIX}/.config/.sommelier.env
+        EOT0
```

Comment:
Not needed if added to `#{CREW_PREFIX}/etc/env.d`?

Reply:
This is for local user modifications to the config, so we don't want to put those into config files in env.d that the user expects to be overwritten.

The move to .config is because the HOME dir has noexec set in newer ChromeOS milestones...

Reply:
Makes sense for overrides.